examples/flutter_gallery/lib/gallery/demos.dart

```diff
@@ -510,3 +553,32 @@ final Map<GalleryDemoCategory, List<GalleryDemo>> kGalleryCategoryToDemos =
       return kAllGalleryDemos.where((GalleryDemo demo) => demo.category == category).toList();
     },
   );
+
+final Map<String, Widget> kDemoDocumentationMaterialButton =
```

Comment:
This seems like an odd way to provide the documentation button widgets. Why not:

```
StatelessWidget MaterialDemoDocumentationButton extends StatelessWidget {
  const MaterialDocumentationButton({
    @required String routeName,
  }) : assert(routeName != null), super(key: key);

  @override
  Widget build(BuildContext context) {
    GalleryDemo demo = kAllGalleryDemos.firstWhere((GalleryDemo demo) => demo.routeName == routeName);
    return IconButton(
      icon: const Icon(Icons.library_books),
      onPressed: () => launch(demo.documentationUrl, forceWebView: true)
    );
  }
}
```

Reply:
SG. Done.